pennylane/interfaces/__init__.py

```diff
@@ -12,52 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Classical interfaces overview
-=============================
-
-**Module name:** :mod:`pennylane.interfaces`
-
 .. currentmodule:: pennylane.interfaces
 
-PennyLane now provides experimental support for additional classical
-machine learning interfaces, specifically PyTorch and TensorFlow eager execution mode.
+PennyLane provides support for various classical
+machine learning interfaces, including Autograd/NumPy, PyTorch,
+and TensorFlow.
```

Comment:
The interfaces have access to gradients of a QNode, and can therefore integrate quantum computations in their pipelines for machine learning and optimization.

Reply:
```suggestion
and TensorFlow. The interfaces have access to gradients of a QNode, and
can therefore integrate quantum computations into the machine learning-library
pipeline for machine learning and optimization.
```